README.md

```diff
@@ -53,7 +53,7 @@ console.log('caught them all')
 
 ## ReadStream
 
-Create a read-promise stream. Pass it a function that takes the number of bytes or objects of wanted data and and uses `this.push` or `return` to push values or promises. This function should return a promise that indicates when the object/chunk are fully processed. Return or push `null` to end the stream.
+Creates a read-promise stream which accepts a function that takes the number of bytes or objects of wanted data as arguments and uses `this.push` or `return` to push values or promises. This function should return a promise that indicates when the object/chunk is fully processed. Return or push `null` to end the stream.
```

Comment:
should these be plural? ```objects/chunks are```

Reply:
I don't think so since it's talking about the function that handles each object/chunk, yeah?